includes/class-wc-payments-order-service.php

```diff
@@ -752,6 +773,37 @@ public function get_fraud_meta_box_type_for_order( $order ) : string {
 	/**
 	 * Given the payment intent data, adds it to the given order as metadata and parses any notes that need to be added
 	 *
+	 * @param WC_Order                                                          $order The order.
+	 * @param WC_Payments_API_Payment_Intention|WC_Payments_API_Setup_Intention $intent The payment or setup intention object.
+	 *
+	 * @throws Order_Not_Found_Exception
+	 */
+	public function attach_intent_info_to_order( WC_Order $order, $intent ) {
+		// first, let's prepare all the metadata needed for refunds, required for status change etc.
+		$intent_id              = $intent->get_id();
+		$intent_status          = $intent->get_status();
+		$payment_method         = $intent->get_payment_method_id();
+		$customer_id            = $intent->get_customer_id();
+		$currency               = $intent instanceof WC_Payments_API_Payment_Intention ? $intent->get_currency() : $order->get_currency();
+		$charge                 = $intent instanceof WC_Payments_API_Payment_Intention ? $intent->get_charge() : null;
+		$charge_id              = $charge ? $charge->get_id() : null;
+		$payment_transaction    = $charge ? $charge->get_balance_transaction() : null;
+		$payment_transaction_id = $payment_transaction ? $payment_transaction['id'] : '';
```

Comment:
```suggestion
		$payment_transaction_id = $payment_transaction['id'] ?? '';
```
Maybe can be written as ^